views/programmer.hbs

```diff
@@ -51,5 +56,12 @@
 <script>
 $(Paysage.programmerInit);
 </script>
+<script>
```

Comment:
I'd like to keep a single `<script>` tag here if possible. It looks like this code could be moved into the `Paysage.programmerInit` function, keeping all initialization together.